.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+jdk:
```

Comment:
I think I should be able to add it without causing any issues on this branch. In the worst case it'll probably just require you to rebase. I'd like to have Travis enabled so that it can run the tests on this branch without me having to pull it down.